frontend/__tests__/providers/toast-context.test.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import ToastContext from "../../providers/contexts/ToastContext";
+
+describe("Toast context", () => {
+  it("should render toast context that is not null", () => {
+    expect(<ToastContext />).not.toBeNull();
+  });
+});
```

Comment:
The toast files that were provided to you do not need to be tested, instead they should be set up to be ignored by jest.